x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/find_threshold_signals.test.ts

```diff
@@ -99,11 +101,19 @@ describe('findThresholdSignals', () => {
     expect(mockSingleSearchAfter).toHaveBeenCalledWith(
       expect.objectContaining({
         aggregations: {
-          'threshold_0:host.name': {
-            terms: {
-              field: 'host.name',
-              min_doc_count: 100,
+          thresholdTerms: {
+            composite: {
               size: 10000,
+              after: undefined,
```

Comment:
The

```ts
after: undefined
```

in the tests in this file also become

```ts
after: null
```

if `buildThresholdMultiBucketAggregation()` is [updated](https://github.com/elastic/kibana/pull/131985/files#r924737563)